charts/orchestrator-k8s/Chart.lock

```diff
@@ -0,0 +1,12 @@
+dependencies:
+- name: backstage
+  repository: https://janus-idp.github.io/helm-backstage
```

Comment:
IIRC, this one is obsolete and should be replaced with https://redhat-developer.github.io/rhdh-chart/